### ISSUE TITLE
[Messenger] Undefined constant "SIGTERM"

### ISSUE THREAD

**Issue by mstramek-CKPL** [symfony/symfony#51863]
### Symfony version(s) affected

6.3.5

### Description

It looks like pcntl dependency is not optional again.

### How to reproduce

When trying to run `messenger:consume xyz`:
```
In ConsumeMessagesCommand.php line 261:
[Error]
Undefined constant "SIGTERM"
```

### Possible Solution

Check whether constants are defined in ConsumeMessagesCommand.php line 261

**Comment by wouterj** [comment]
duplicate of https://github.com/symfony/symfony/issues/51836

**Comment by m-stramek** [comment]
> duplicate of #51836

But in #51836 is info:

> Duplicate of: https://github.com/symfony/symfony/issues/51790, fixed in: https://github.com/symfony/symfony/pull/51791

https://github.com/symfony/symfony/pull/51791 solved problem for version < 6.3, but it reappeared in 6.5
